File: multi-lang/java/Grade.java
public class Grade {
    public static void main(String[] args) {
        int score = 85;
        String grade;
        if (score >= 90) grade = "A";
        else if (score >= 80) grade = "B";
        else if (score >= 70) grade = "C";
        else grade = "F";

        for (int i = 1; i <= 3; i++) {
            System.out.println("Attempt " + i + ": Score=" + score + " Grade=" + grade);
        }
    }
}

File: multi-lang/php/grade.php
<?php
$score = 85;
$grade = $score >= 90 ? "A" : ($score >= 80 ? "B" : ($score >= 70 ? "C" : "F"));

for ($i = 1; $i <= 3; $i++) {
    echo "Attempt $i: Score=$score Grade=$grade\n";
}
?>